src/cache/cache_las.c

```diff
@@ -855,7 +851,12 @@ __wt_las_insert_updates(WT_CURSOR *cursor, WT_BTREE *btree, WT_PAGE *page, WT_MU
     if (ret != 0)
         for (i = 0, mementop = mementos->mem; i < mementos_cnt; i++, mementop++)
             __wt_buf_free(session, &mementop->key);
+    if (modify_value != NULL)
```

Comment:
Add a comment indicating why is the special handling required i.e. why NULL check?

Reply:
Added.